src/Screens/components/Button.tsx

```diff
@@ -29,6 +30,7 @@ const styles = RN.StyleSheet.create({
   container: {
     alignSelf: 'stretch',
     borderRadius,
+    ...shadow(7),
```

Comment:
this shadow style abstracts away ios and android differences in shadow handling.